client/my-sites/plans/ecommerce-trial/wooexpress-plans/style.scss

```diff
@@ -16,6 +16,12 @@
 			max-width: 100%;
 			width: 100%;
 		}
+
+		@media (max-width: 375px) {
```

Comment:
Is this the right breakpoint? I feels like it might need to be wider to handle German and/or other translations.

Reply:
Hmm, that's a good point. The original issue was only happening on breakpoints much smaller than `$break-mobile` but that example was also English.

To accommodate languages like German, it's probably better to move this selector into the one above it that's targeting `$break-mobile` (which, I think, is 480px).